### ISSUE TITLE
After using the plugin for a while the TIPS window keeps coming up

### ISSUE THREAD

**Issue by ImpulseMPD** [nguyenquyhy/Flight-Tracker-StreamDeck#97]
Hi Guys, 
after some use I found that after clicking level change several times for example dropping 100ft the sim starts to show the TIPS screen and it doesn't go away.
![image](https://user-images.githubusercontent.com/55635517/102361031-de63dc80-4006-11eb-9217-c3e975c539e4.png)

**Comment by linmar22** [comment]
Can confirm. Some more info - 
This happens when holding down one of the keys for a while, like in/decreasing the AP ALT.
At some point the plugin or underlying SimConnect bridge crashes or disconnects from the sim.
My theory is that while you're holding down the key, the plugin spams SimConnect, which tries to reconnect on every key input, causing a near-never-ending stream of connection notifications.

**Comment by nguyenquyhy** [comment]
Your observation is mostly correct, but let me give some more details here:
- When you hold down the key, the plugin sends repeated events to SimConnect, but I wouldn't call it spamming when the rate is only once every 400ms. There shouldn't be any hard limit on that either since I have even tried to send at a much higher rate and the issue happens randomly.
- When this issue happens, SimConnect returns an Unknown error. The connection between the plugin and SimConnect is intact, but SimConnect stops sending data or responding to events or even sending further errors. The only solution I can found in that situation is to force a reconnect to SimConnect.
- It gets worse when the reconnection succeeds, but SimConnect is still in some weird state. It will send an Unknown error and then again ignore the plugin. The plugin attempts a reconnection again.
- This seems to go away once the plugin is restarted, so previously I thought SimConnect need more time between reconnection to settle its issue, and I added a 2s delay between reconnection attempts. This seems to help me, but doesn't help others.

Anyway, this is a duplicate of #56, so let me close this one for now.

**Comment by linmar22** [comment]
Nice - is displaying the tooltip done by simconnect or the plugin? Because one of the solutions would just be to check when the last "Connected to Stream Deck plugin" tooltip was displayed and suppress any new ones until the timer expires. That way they can still do the reconnection dance in the background without bothering the user too much. Something like once every 5s would make it a lot less "in your face". Realize its a hack, but hey...
